ksql-functional-tests/src/main/java/io/confluent/ksql/test/tools/TestExecutorUtil.java

```diff
@@ -137,8 +121,7 @@ private TestExecutorUtil() {
           serviceContext.getSchemaRegistryClient());
       testCase.setGeneratedTopologies(
           ImmutableList.of(persistentQueryMetadata.getTopologyDescription()));
-      testCase.setGeneratedSchemas(
-          ImmutableList.of(persistentQueryMetadata.getSchemasDescription()));
+      testCase.setGeneratedSchemas(persistentQueryMetadata.getSchemasDescription());
```

Comment:
How come the `ImmutableList.of()` call was removed?

Reply:
Moved it to the setter